teacher.rb

```diff
@@ -1,49 +1,14 @@
-class Teacher
-  attr_reader :age, :salary, :phase, :performance_rating, :target_raise
-  attr_accessor :name
+require_relative 'staff'
 
-  def initialize(options={})
-    @phase = 3
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
-    @target_raise = 1000
-  end
+class Teacher < Staff
 
-  def offer_high_five
-    "High five!"
-  end
+  attr_reader :performance_rating
 
-  def set_phase(num)
-    @phase = num
-    "Cool, I've always wanted to teach phase #{num}!"
-  end
-
-  def teach_stuff
-    response = ""
-    response += "Listen, class, this is how everything works, fo SHO! "
-    response += "*drops flat-out insane knowledge bomb* "
-    response += "... You're welcome. *saunters away*"
-    response
-  end
-
-  def salary=(new_salary)
-    puts "This better be good!"
-    @salary = new_salary
-  end
-
-  def receive_raise(raise)
-    @salary += raise
+  def initialize(options={})
+    super
+    @target_raise = 1000
```

Comment:
You can pull this out into a constant, set the ```@target_raise ``` attribute in the super class. Sweeeeet stuff!